docs/cri/development.md

```diff
@@ -0,0 +1,53 @@
+# Getting Started for Developers
+
+This document provides steps to build `cri` plugin and test it locally.
+
+## Install Dependencies
+
+1. Follow this [guide](https://github.com/containerd/containerd/blob/master/BUILDING.md) and install required dependencies.
+2. Install [CNI](https://github.com/containernetworking/cni) by following its homepage.
```

Comment:
see `sudo make install-cri-deps`

Reply:
I was aware of it but it doesn't work on local machine for me. I asked the question in slack #containerd-dev. It seems these [scripts](https://github.com/containerd/containerd/tree/master/script/setup)  are supposed to be used by CI for now. Whenever it is good to run on local machine, we can update this step.

Reply:
yeah that's a bit easier.. sry power is up and down here snow/ice storm.. loosing track of all the slack channel threads

Reply:
This way is also good, maybe in some note section you can point `make install-cri-deps`

Reply:
ok, I don't have time to test `make install-cri-deps`. I will put it to the notes.